composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": ">=5.3.0",
-	"illuminate/support": "~4"
+	"laravel/framework": ">=8"

```

Comment:
It's been a real long time since I've looked at laravel stuff (just because I've not needed to use it), but I'm 90% certain we shouldn't depend on laravel/framework and instead should depend on `illuminate/support`.